### PR TITLE
chore(op-test-vectors): Update `ExecutionFixture` to use `op-alloy-consensus` types 

### PR DESCRIPTION
**Pull request by 0xKitsune**
This PR addresses #40, replacing the all `anvil` types in the `ExecutionFixture` struct with types from `op-alloy-consensus` . Additonally `opt8n` was updated to reflect these changes.

```rust
pub struct ExecutionFixture {
    /// The execution environment sets up the current block context.
    pub env: ExecutionEnvironment,
    /// The initial state of the accounts before running the transactions, also called the
    /// "pre-state".
    pub alloc: HashMap<Address, AccountState>,
    /// The expected state of the accounts after running the transactions, also called the
    /// "post-state".
    pub out_alloc: HashMap<Address, AccountState>,
    /// Transactions to execute.
    #[serde(rename = "txs")]
    pub transactions: Vec<OpTypedTransaction>,
    /// The expected result after executing transactions.
    pub result: ExecutionResult,
}
```

### CODE REVIEW

File: bin/opt8n/src/opt8n.rs [hunk 1]
```diff
@@ -183,6 +201,94 @@ impl Opt8n {
     }
 }
 
+// TODO: Consider adding `From` implementation for
```
Comment (refcell) [inline bin/opt8n/src/opt8n.rs:204]: Agree - I will add issues in op-alloy for this.

File: crates/op-test-vectors/src/execution.rs [hunk 2]
```diff
@@ -159,53 +98,4 @@ mod tests {
             .expect("failed to parse expected result");
         assert_eq!(serialized_value, expected_value);
     }
-
```
Comment (refcell) [inline crates/op-test-vectors/src/execution.rs:162]: Can we add a test/tests here for deserializing the op types?

File: bin/opt8n/src/opt8n.rs (hunk 1 above)
Comment (0xKitsune) [reply bin/opt8n/src/opt8n.rs:204]: Sounds good, I can open a PR for these issues on `op-alloy` later today/tomorrow.

**Review by refcell — APPROVED** [review]
Approved, let's just add a few tests for deserialization

**Comment by clabby** [comment]
Merging to unblock.

**Pull request merged**